src/dodal/common/beamlines/beamline_utils.py

```diff
@@ -22,7 +24,6 @@
 
 ACTIVE_DEVICES: dict[str, AnyDevice] = {}
 BL = ""
```

Comment:
```suggestion
BL = ""
PATH_PROVIDER: PathProvider = StaticPathProvider(UUIDFilenameProvider(), "/tmp")
```

Gives us a sensible default when the PathProvider hasn't been explicitly set (by blueapi presumably).

Reply:
Currently I _think_ every beamline module requires one and has one set? Although I guess this won't do harm?

Reply:
Although it changes the behavior of dodal somewhat by having it. I think I prefer to leave it and still demand you set it yourself.

Reply:
Opened a discussion her https://github.com/DiamondLightSource/dodal/issues/1129. It is an problem for another issue.